code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -284,6 +285,11 @@
 		else
 			disky.melee_attack_chain(src, attacked_target)
 		return
+
+	if(istype(attacked_target, /obj/machinery/nuclearbomb))
+		var/obj/machinery/nuclearbomb/nuke = attacked_target
+		nuke.ui_interact(src)
```

Comment:
```suggestion
		nuke.ui_interact(src)
		audible_message("You hear something smacking softly against the nuke!")
```

Reply:
maybe instead of the nuke, do a pointer for the nuke